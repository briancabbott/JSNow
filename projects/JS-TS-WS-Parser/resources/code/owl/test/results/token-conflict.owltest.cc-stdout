input (0 - 3)
  number - 1.000000 (0 - 3)
