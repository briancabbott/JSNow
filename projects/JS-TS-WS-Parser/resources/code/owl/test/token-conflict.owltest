1.0
---
.token digit '1'
input = number | digit
