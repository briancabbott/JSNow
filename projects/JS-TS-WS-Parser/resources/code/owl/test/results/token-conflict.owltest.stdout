. 1.0   
  input 
